### ISSUE TITLE
`collect_river_data` for ECCC rivers fails with `UnicodeDecodeError`

### ISSUE THREAD

**Issue by douglatornell** [SalishSeaCast/SalishSeaNowcast#155]
Failures started on 21-Feb-2023.

Traceback:
```python-traceback
Traceback (most recent call last):
  File "/SalishSeaCast/NEMO_Nowcast/nemo_nowcast/worker.py", line 391, in _do_work
    checklist = self.worker_func(
  File "/SalishSeaCast/SalishSeaNowcast/nowcast/workers/collect_river_data.py", line 103, in collect_river_data
    day_avg_discharge = day_avg_discharge_funcs[data_src](river_name, data_date, config)
  File "/SalishSeaCast/SalishSeaNowcast/nowcast/workers/collect_river_data.py", line 127, in _calc_eccc_day_avg_discharge
    df = pandas.read_csv(
  File "/SalishSeaCast/nowcast-env/lib/python3.10/site-packages/pandas/util/_decorators.py", line 211, in wrapper
    return func(*args, **kwargs)
  File "/SalishSeaCast/nowcast-env/lib/python3.10/site-packages/pandas/util/_decorators.py", line 331, in wrapper
    return func(*args, **kwargs)
  File "/SalishSeaCast/nowcast-env/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 950, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/SalishSeaCast/nowcast-env/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 605, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/SalishSeaCast/nowcast-env/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 1442, in __init__
    self._engine = self._make_engine(f, self.engine)
  File "/SalishSeaCast/nowcast-env/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 1753, in _make_engine
    return mapping[engine](f, **self.options)
  File "/SalishSeaCast/nowcast-env/lib/python3.10/site-packages/pandas/io/parsers/c_parser_wrapper.py", line 79, in __init__
    self._reader = parsers.TextReader(src, **kwds)
  File "pandas/_libs/parsers.pyx", line 547, in pandas._libs.parsers.TextReader.__cinit__
  File "pandas/_libs/parsers.pyx", line 636, in pandas._libs.parsers.TextReader._get_header
  File "pandas/_libs/parsers.pyx", line 852, in pandas._libs.parsers.TextReader._tokenize_rows
  File "pandas/_libs/parsers.pyx", line 1965, in pandas._libs.parsers.raise_parser_error
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xe9 in position 81: invalid continuation byte
```

**Comment by douglatornell** [comment]
Changing `pandas.read_csv()` from default utf-8 encoding to use `encoding=iso-8859-1` resolved the problem. So, I suspect that this is an unintended change in the ECCC code that generates the .csv files because there is no sensible reason to revert from utf-8 to iso-8859-1.

**Comment by douglatornell** [comment]
Hacked `collect_river_data` on `skookum` to add `encoding=iso-8859-1`  arg. Waiting to see if ECCC reverts encoding change before changing our code here.

**Comment by douglatornell** [comment]
On 27-Feb-2023 `collect_river_data ECCC ...` again failed. This time with a `pandas` unmatched column label exception for the `Discharge / Débit` column. That suggests that the change to iso8859-1 encoding that caused this issue was reverted in the ECCC code. 

I reverted my `encoding=iso-8859-1`  arg hack on `skookum`, and `collect_river_data` ran successfully. So, I'm closing this issue because no code change is required in this repo.
